### PR TITLE
[Barefoot] enable bulk api`s in SAI

### PR DESCRIPTION
**Pull request by KostiantynYarovyiBf**
Why I did it:    
    Bulk was disable
How I did it:
    enable bulk api`s in SAI for Barefoot
How to verify it:
    Run sanity regression tests

### CODE REVIEW

**Review by kcudnik — APPROVED** [review]

**Comment by kcudnik** [comment]
entire PR is just adding empty line, is that it ?

**Comment by KostiantynYarovyiBf** [comment]
> entire PR is just adding empty line, is that it ?

this was a test.  I decided to check if my changes affect the fail CI tests. Now I returning my changes

**Review by kcudnik — DISMISSED** [review]

File: syncd/VendorSai.cpp [hunk 1]
```diff
@@ -616,7 +616,7 @@ sai_status_t VendorSai::bulkCreate(
             ptr = m_apis.lag_api->create_lag_members;
             break;
 
-        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER:
+        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP:
```
Comment (kcudnik) [inline syncd/VendorSai.cpp:619]: this should be SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER

File: syncd/VendorSai.cpp [hunk 2]
```diff
@@ -675,7 +675,7 @@ sai_status_t VendorSai::bulkRemove(
             ptr = m_apis.lag_api->remove_lag_members;
             break;
 
-        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER:
+        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP:
             ptr = m_apis.next_hop_group_api->remove_next_hop_group_members;
```
Comment (kcudnik) [inline syncd/VendorSai.cpp:679]: this should be SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER

**Review by kcudnik — CHANGES_REQUESTED** [review]
address comments

File: syncd/VendorSai.cpp (hunk 2 above)
Comment (KostiantynYarovyiBf) [reply syncd/VendorSai.cpp]: ok, I reverted this commit

**Review by kcudnik — DISMISSED** [review]

**Review by kcudnik — APPROVED** [review]

**Pull request merged**